Maths/fibonacci2.py

```diff
@@ -0,0 +1,26 @@
+# Fibonacci Sequence Using Recursion
+
+
+def fibonacci(limit):
+    first_number = 1
+    second_number = 1
+
+    fibonaccci = [first_number , second_number ]
+
+    for i in range(limit):
+        first_number , second_number = second_number , first_number + second_number
+
+        fibonaccci.append(second_number)
+    print(fibonaccci)
```

Comment:
Look at the output of your script:
```python
How many terms to include in fibonacci series: 5
The first 5 terms of the fibonacci series are as follows
[0, 1, 1, 2, 3, 5, 8]
```
Now, The length of the printed array should be **5**.